### PR TITLE
ftux deployments by app

### PR DESCRIPTION
**Pull request by neurosnap**
Fixes:
- Deployments page now has same status pill as status page
- Deployments page now has correct "finish setup" link
- Deployments are listed by `App` instead of `Account`
- If there are 0 operations for an `Account` we resolve that status to `unknown` instead of `succeeded`

# BEFORE
Screenshot 2023-05-06 at 3 24 51 PM

# AFTER
Screenshot 2023-05-06 at 3 24 55 PM

### CODE REVIEW

File: src/deploy/operation/index.ts [hunk 1]
```diff
@@ -273,6 +273,11 @@ export const fetchEnvOperations = api.get<EnvOpProps>(
   "/accounts/:envId/operations?page=:page",
 );
 
+export const fetchAllEnvOps = thunks.create<EnvIdProps>(
+  "fetch-all-env-ops",
+  combinePages(fetchEnvOperations, { max: 5 }),
```
Comment (madhuravius) [inline src/deploy/operation/index.ts:278]: we might want to keep fetching if the most recent ops are backups, that thing eats up a lot of pages even after just a week or two i think

**Review by madhuravius — DISMISSED** [review]

File: src/deploy/app/index.ts [hunk 2]
```diff
@@ -166,6 +170,24 @@ export const selectAppsForTableSearch = createSelector(
...
+  selectEnvironments,
+  selectAppsAsList,
+  (envs, apps) => {
+    return apps.filter((app) => {
+      const env = findEnvById(envs, { id: app.environmentId });
+      if (!hasDeployEnvironment(env)) {
+        return false;
+      }
+      if (env.onboardingStatus === "unknown") {
+        return false;
```
Comment (madhuravius) [inline src/deploy/app/index.ts:183]: we might want to leave a note somewhere that we only will show environments deployed with our new flow (aka legacy/v1 is not displayed on this list)

File: src/deploy/operation/index.ts (hunk 1 above)
Comment (neurosnap) [reply src/deploy/operation/index.ts:278]: Yeah I'm just worried about over-fetching.  I can increase it to 10 pages?

File: src/deploy/app/index.ts (hunk 2 above)
Comment (neurosnap) [reply src/deploy/app/index.ts:183]: Good idea

File: src/deploy/operation/index.ts (hunk 1 above)
Comment (madhuravius) [reply src/deploy/operation/index.ts:278]: we can leave it at 5 but maybe make a ticket for us to have a paginator that just keeps going and collects until X non backup operations found?

Comment (madhuravius) [reply src/deploy/operation/index.ts:278]: or 5 instantly and come back somehow in a throttled way AFTER 5 until it finds something?

**Review by madhuravius — APPROVED** [review]

**Pull request merged**